### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by Mboudali** [sandeepmistry/arduino-nRF5#511] [General]
Hi there, 
As per the title, is this project abandoned?
I just started tinkering with nrf52 SOCs. I am facing some difficulties getting the BLE functionality working.
Hardware: Ebyte nrf52833, St-Link V2

**Comment by Adix3** [comment]
Hi.
Have you solved your problem? What is it actually about?

**Reply by Mboudali** [reply]
Hi Adix3, 
Yes, I fixed the problem. I had to write a variant for the nRF52833 and download the right softdevice from the nRF website.

**Reply by Borkuz** [reply]
Can you share your variant?

**Reply by Mboudali** [reply]
Not sure how I can add it to this repository... If someone could guide me, that would be great.
https://github.com/Mboudali/arduino_ebyte73

**Comment by Borkuz** [comment]
I think the main problem is that those modules don't come with a (correct) bootloader. So you need to first write the Adafruit nrf52 bootloader and then use the generic nrf52 board config and then it should work. But I'm no expert so anyone (@sandeepmistry) correct me if I'm wrong.
